maps/models.py

```diff
@@ -314,7 +331,6 @@ class Map(models.Model):
         related_name="roads_source_for",
         null=True, blank=True,
     )
-
```

Comment:
hmm, I would prefer these new lines to stay here for group-readability. I don't think that breaches PEP8 in any way.